search-sentiments-map/routes/sentiment.js

```diff
@@ -0,0 +1,48 @@
+/** Server-side script that uses the Google Cloud Natural Language API to get the sentiment score. */
+
+const express = require('express');
+var router = express.Router();  // Using Router to divide the app into modules.
+const language = require('@google-cloud/language');
+
+var bodyParser = require('body-parser');  
+// Create application/x-www-form-urlencoded parser.
+var urlencodedParser = bodyParser.urlencoded({ extended: false })  
+router.post('/', urlencodedParser, (req, res) => { 
+  const searchTopic = req.body.searchTopic;
+  console.log('ntarn debug: '+ searchTopic);
+  quickstart(searchTopic).then(score => {
+    // Prepare output in JSON format.  
+    response = {  
+      sentimentScore:score,  
+    };  
+    console.log('ntarn debug: searchTopic score' + response.sentimentScore);  
+    res.end(JSON.stringify(response));
+  }).catch(err => {
+    console.log(err);
+  })
+});
+
+async function quickstart(searchTopic) {
```

Comment:
Rename this to something more relevant.
Note: You're also returning the sentiment score.